packages/nodes-base/nodes/Airtop/actions/window/create.operation.ts

```diff
@@ -9,15 +9,11 @@ import { NodeApiError } from 'n8n-workflow';
 import { validateAirtopApiResponse, validateSessionId, validateUrl } from '../../GenericFunctions';
 import { apiRequest } from '../../transport';
 import type { IAirtopResponse } from '../../transport/types';
+import { urlField } from '../common/fields';
 
 export const description: INodeProperties[] = [
 	{
-		displayName: 'URL',
-		name: 'url',
-		type: 'string',
-		default: '',
-		placeholder: 'https://google.com',
-		description: 'Initial URL to load in the window. Defaults to http://google.com.',
```

Comment:
I would keep the description here so it gets overwritten with more details on the default if left empty.